Creating User Section

vultuk opened the pull request:
The user section is being designed and created, including States, Controllers and all
appropriate templates.

Commit:
Basic Page Header

Basic Page header has been designed to include a heading, an aside
and some right aligned option buttons. The user list page has also
had a table added to it and the add user state has been added.

review-ninja commented:
<a href="https://app.review.ninja/vultuk/vicidial-client/pull/1" target="_blank"><img src="https://app.review.ninja/59860300/pull/1/badge" alt="ReviewNinja"/>

Commit:
Changes to Layouts and Back End Code

Updated Grunt to allow the following
- Import of jQuery
- Import of Bootstrap JS files
- Copying of Bootstrap fonts to appropriate directory

Changed Users section
- Added temporary array of users to Controller
- Created table layouts that work with ng-repeat
- Streamlined the menu by removing un-needed links

The URL structure was also changed meaning that the admin area
is always prefixed by /admin
